### PR TITLE
fix: change titleCase to pascalCase

### LINKED ISSUE #2396: When using withHooks we seeing a space as `use<queryName>Lazy Query` instead of one word
**Describe the bug**
 

The generated code has a `space` like below.
```
      export function useTodoStateLazy Query(baseOptions?: ApolloReactHooks.LazyQueryHookOptions<SchemaTypes.ITodoStateQuery, SchemaTypes.ITodoStateQueryVariables>) {
        return ApolloReactHooks.useLazyQuery<SchemaTypes.ITodoStateQuery, SchemaTypes.ITodoStateQueryVariables>(TodoStateDocument, baseOptions);
      };

```

**To Reproduce**

3. My `codegen.yml` config file:

```yml
  models.tsx:
    documents: "packages/files-client-state/src/**/*.gql"
    schema: 
      - ./packages/client-state/src/**/*.graphql
      - ./packages-modules/module/server/src/graphql/**/*.graphql
    config:
        namingConvention: keep
        typesPrefix: I
        withMutationFn: true
        withHOC: true
        withComponent: true
        withHooks: true
    preset: "@cdmbase/import-types-preset"
    presetConfig:
      typesPath: "core"
      importTypesNamespace: SchemaTypes
```

**Expected behavior**

```
      export function useTodoStateLazy Query(baseOptions?: ApolloReactHooks.LazyQueryHookOptions<SchemaTypes.ITodoStateQuery, SchemaTypes.ITodoStateQueryVariables>) {
        return ApolloReactHooks.useLazyQuery<SchemaTypes.ITodoStateQuery, SchemaTypes.ITodoStateQueryVariables>(TodoStateDocument, baseOptions);
      };

```
**Environment:**

- OS: 
- `@graphql-codegen/...`:  1.6.1
- NodeJS: 

**Additional context**

### PR DESCRIPTION
**Pull request by benjie**
Fixes #2396

`titleCase` adds spaces at word boundaries; we do not want spaces in any of these places. This PR changes `titleCase` usage to `pascalCase` to match the rest of the codebase.

```
$ node
> c = require('change-case')
> c.titleCase('LazyQuery')
'Lazy Query'
> c.pascalCase('LazyQuery')
'LazyQuery'
> c.pascalCase('lazy query')
'LazyQuery'
> 
```

_(Most of the uses of `titleCase` currently work because they run against a single word (`operationType`) so no spaces are added; however keeping these opens the door to copy/paste errors so I think it's safer to eradicate `titleCase`.)_

### CODE REVIEW

File: packages/plugins/typescript/react-apollo/src/visitor.ts [hunk 1]
```diff
@@ -174,7 +174,7 @@ export class ReactApolloVisitor extends ClientSideBaseVisitor<ReactApolloRawPlug
 
     if (operationType === 'Query') {
       const lazyOperationName: string = this.convertName(node.name.value, {
-        suffix: titleCase('LazyQuery'),
+        suffix: pascalCase('LazyQuery'),
```
Comment (benjie) [inline packages/plugins/typescript/react-apollo/src/visitor.ts]: I've kept this call here even though it's an expensive no-op to aid future copy-pasting. We could simplify this to:

```suggestion
        suffix: 'LazyQuery',
```

**Comment by hrg921** [comment]
Thanks for the nice fix, I was suffering this issue.

**Comment by dotansimha** [comment]
Thank you @benjie ! This is awesome. Any chance you can rebase and resolve conflicts?

**Comment by benjie** [comment]
@dotansimha I’m away so I’ve attempted to do so from my phone; hopefully it’s all good but if not let me know and I can fix it from my computer next week.

**Pull request merged**